tests/mod_test.go

```diff
@@ -341,4 +341,53 @@ func TestMods(t *testing.T) {
 			}
 		})
 	}
+
+	t.Run("Prevent Mod Reference Matching Existing Mod ID", func(t *testing.T) {
+		// First create a mod to get its ID
+		token, _, err := makeUser(ctx)
+		testza.AssertNoError(t, err)
+
+		// Create the first mod without tags to avoid validation issues
```

Comment:
Tags (presumably ficsit.app tags?) don't seem to come into play with this test?
```suggestion
		// Create the first mod
```